### PR TITLE
Invitee response status changes

### PR DESCRIPTION
**Pull request by christophermark**
Adds support for the new invitee response statuses:
- `NONE` is deprecated
- `UNKNOWN` is added
- `DELEGATED` is added

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/Invitee.java [hunk 1]
```diff
@@ -27,12 +27,20 @@
     public enum ResponseStatus {
 
         /**
+         * [Deprecated]
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Invitee.java]: Move this below the description and use javadoc. 

`@deprecated Use `UNKNOWN` instead.`

**Comment by christophermark** [comment]
Doc fixed

File: src/main/java/com/robinpowered/sdk/model/Invitee.java [hunk 2]
```diff
@@ -28,11 +28,19 @@
...
          */
+        @Deprecated
         @SerializedName("none")
         NONE("none"),
 
         /**
+         * The person's response is unknown.
+         */
+        @SerializedName("unkonwn")
+        UNKNOWN("unkonwn"),
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Invitee.java]: sp

**Comment by christophermark** [comment]
Fixed and squashed!

File: src/main/java/com/robinpowered/sdk/model/Invitee.java (hunk 2 above)
Comment (christophermark) [reply src/main/java/com/robinpowered/sdk/model/Invitee.java]: This never happened! 🙈

**Review by brianmuse — APPROVED** [review]

**Pull request merged**